packages/core/src/agent/base.ts

```diff
@@ -101,16 +97,6 @@ export function createTaskOutputStream<
   });
 }
 
-export type AgentStreamChatResponse<Options extends object> = {
-  response: ChatResponseChunk<Options>;
-  sources: ToolOutput[];
-};
-
-export type AgentChatResponse<Options extends object> = {
```

Comment:
`AgentChatResponse` is better than `Response` but most people are using normal chat engines I assume, so we better improve the existing `Response` class (see above)